unit-tests/src/main/scala/java/io/FileInputStreamSuite.scala

```diff
@@ -62,4 +62,12 @@ object FileInputStreamSuite extends tests.Suite {
     assert(fis.read() == 0xFF)
     assert(fis.read() == -1)
   }
+
+  testFails("throws proper FileNotFoundException on invalid file names", 765) {
+    assertThrows[FileNotFoundException] { 
+      val fis = new FileInputStream("/the/path/does/not/exist/for/sure")
+      fis.read
```

Comment:
You don't need this line. On the JVM, it will throw a `FileNotFoundException` in the constructor.